cmd/cloud-network-config-admission-controller/main.go

```diff
@@ -0,0 +1,156 @@
+package main
```

Comment:
It would be good if there was a README in this folder with a quick overview. Doesn't need to be too big, just a paragraph explaining what is being checked.

Reply:
I added a doc bloc in this file for the `AdmissionFunc`, hopefully that is fine.